echo/server/tests/common.py

```
@@ -56,6 +58,7 @@ def create_conversation_chunk(
 
 
 def delete_conversation_chunk(conversation_chunk_id: str):
+    logger.debug(f"Deleting conversation chunk: {conversation_chunk_id}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)